plugins/modules/nmcli.py

```diff
@@ -68,9 +68,10 @@
             - Type V(vpn) is added in community.general 5.1.0.
             - Using V(bond-slave), V(bridge-slave), or V(team-slave) implies V(ethernet) connection type with corresponding O(slave_type) option.
             - If you want to control non-ethernet connection attached to V(bond), V(bridge), or V(team) consider using O(slave_type) option.
+            - Type V(loopback) is added in community.general 8.1.0.
```

Comment:
Can you move this up so it fits into the (mostly) sorted list of when other types were added (i.e. between `gsm` and `macvlan`)?

Reply:
Sure. Done